_sass/elements/_ol-ul.scss

```diff
@@ -0,0 +1,3 @@
+ul:not([class]) {
```

Comment:
I really like how you are separating styles to make it easy to read. It might be nice to have an `elements.scss` file with all of these selector styles instead of having separate files for all of them. That would help the project be maintainable and scalable!

Reply:
ohhh so instead of a folder have one file that has all default styles? that sounds cooler. I was following the scss standard start project file of Sparkbox